libs/ledger-live-common/src/families/solana/tx-fees.ts

```diff
@@ -5,6 +5,9 @@ import { Transaction, TransactionModel } from "./types";
 import { assertUnreachable } from "./utils";
 import { VersionedTransaction as OnChainTransaction } from "@solana/web3.js";
 import { log } from "@ledgerhq/logs";
+import { NetworkError } from "@ledgerhq/errors";
+
+const MAX_RETRIES = 10; // TODO Candidate to put in coin config
```

Comment:
before putting this env, i think i would consider this for backend to fix the stability, don't you think?

Reply:
It's not certain that it's a stability issue, Solana is a bit complex and it may just be "normal" blockchain behaviour.

But I agree, discussion is already ongoing with Solana devs and I've asked that exact question, it's on them to tell whether it's normal backend behaviour or not.

In the meantime this is only a workaround.

Reply:
@gre Got the anwser just now: it's indeed not backend instability, it's a strong requirement to have and use latest block hash when calling the `getFeeForMessage` function. And block creation time on Solana is < 1s, so it easily happens that the block hash is already outdated when calling `getFeeForMessage`...

I'm discussing the best approach, we'll probably go for less retries and a hardcoded fees fallback if it fails too much.